modules/ROOT/edits/mule-core-usage-self.adoc

```diff
@@ -0,0 +1,20 @@
+= Monitor Mule Core Usage With Runtime Fabric on Self-Managed Kubernetes
+
+Anypoint Runtime Fabric on Self-Managed Kubernetes enables you to use your own infrastructure monitoring solution to view metrics, monitor and alert on conditions.
+Your monitoring solution can also be used to count and generate alerts on Mule core usage.
+
+== Counting Mule Application Resources
+
+The following procedures describe how to use your monitoring solution to aggregate resource usage across all Mule application containers on Runtime Fabric
+with Self-Managed Kubernetes. 
+
+. Select all Pods containing a label equal to "MuleApplication"
+. From each pod in the prior step, select the Container Name equal to "/app"
+. Use your monitoring solution to aggregate and plot the CPU Usage and CPU Request across each selected container.
```

Comment:
Either lowercase CPU usage and request or format to clarify whether they are GUI labels or CLI syntax.